hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -122,8 +123,9 @@
      * <p/>
      * By default only mandatory probes are being tracked
      */
-    public static final HazelcastProperty PERFORMANCE_METRICS_LEVEL
-            = new HazelcastProperty("hazelcast.performance.metric.level", ProbeLevel.MANDATORY.name());
+    public static final HazelcastProperty PERFORMANCE_METRICS_LEVEL =
+            new HazelcastProperty(Diagnostics.PREFIX + ".metric.level", ProbeLevel.MANDATORY.name())
```

Comment:
I'm not 100% sure about the usage of `Diagnostics.PREFIX` here to construct the name. This is the source for the reference manual, and now it has a redirection to other code. If @Serdaro is fine with it, I have no objection though.